RecoTracker/LSTCore/src/alpaka/Event.dev.cc

```diff
@@ -264,11 +267,13 @@ void Event::addPixelSegmentToEvent(std::vector<unsigned int> const& hitIndices0,
       alpaka::createSubView(segmentsBuffers_->totOccupancySegments_buf, (Idx)1u, (Idx)pixelModuleIndex);
   alpaka::memcpy(queue_, dst_view_totOccupancySegments, src_view_size);
 
-  auto dst_view_nMDs = alpaka::createSubView(miniDoubletsBuffers_->nMDs_buf, (Idx)1u, (Idx)pixelModuleIndex);
+  MiniDoubletsOccupancy mdsOccupancy = miniDoubletsDC_->view<MiniDoubletsOccupancySoA>();
+  auto nMDs_view = alpaka::createView(devAcc_, mdsOccupancy.nMDs(), (Idx)nLowerModules_ + 1);
```

Comment:
`mdsOccupancy.metadata().size()`; I'm not sure if a `cms` space has a direct method to get a correctly sized view.

Reply:
I couldn't find a way to create an Alpaka view directly. `view<T>()` only returns a pointer, so there's not enough information, but maybe we can ask Andrea or Matti if we missed something.

Reply:
No, you are right: currently we don't have a way to get an `alpaka::View<>` over a SoA column or scalar.
The reason is the one you mention: the SoA knows only the pointers, it does not know the device, so it cannot build a SoA.

It is something I would like to improve, and any suggestions are welcome.